SimpleSDMPolygons/Project.toml

```diff
@@ -0,0 +1,39 @@
+name = "SimpleSDMPolygons"
+uuid = "dd4c243c-bc7b-45f6-b2b3-6bd128d3a12b"
+authors = ["Michael D. Catchen <mdcatchen@gmail.com>"]
+version = "0.1.0"
```

Comment:
`1.0.0` for initial releases so we can do `SimpleSDMPolygons = "1"` in `[compat]`

Reply:
done